.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/vscode/devcontainers/python:0-3.10-bullseye
+FROM mcr.microsoft.com/vscode/devcontainers/python:3.10-bullseye
```

Comment:
Python is now up to 3.12 available, Is there a reason we chose to stay with 3.10?

Reply:
Good point. I do not see a 3.12 official MS container for that flavor and 3.12 But I do see one for 3.11
https://[mcr.microsoft.com/v2/vscode/devcontainers/python/tags/list](https://mcr.microsoft.com/v2/vscode/devcontainers/python/tags/list)

Reply:
Yes, I originally tried with 3.12 but was overwhelmed with errors. 3.11 was much better but then I talked with our Data Science Lead who recommended 3.10.12.

Reply:
ok.. sounds good.